src/main/java/com/plaid/client/model/paymentinitiation/Schedule.java

```diff
@@ -0,0 +1,25 @@
+package com.plaid.client.model.paymentinitiation;
+
+public class Schedule {
+  private String interval;
+  private Integer intervalExecutionDay;
+  private String startDate;
+
+  public Schedule(String interval, Integer intervalExecutionDay, String startDate) {
```

Comment:
I think it's a bit more canonical to use `int` instead of `Integer` for the constructor here

Reply:
ah yea that sounds like a Java thing I learned at some point